query/src/datasources/table/fuse/util/statistic_helper.rs

```diff
@@ -215,3 +217,29 @@ pub fn merge_stats(schema: &DataSchema, l: &Stats, r: &Stats) -> Result<Stats> {
     };
     Ok(s)
 }
+
+pub fn merge_appends(
```

Comment:
Emm, it seems it's time to move this helper to a folder named like statistics, utilvis a good name for others to exploring :)

Reply:
agree ，there is a pending pr https://github.com/datafuselabs/databend/pull/2408 ，shall i merge it into this pr？

Reply:
Oh, pls do not bother reviewing the pr #2480, forgot that the pr accumulated lots of irrelevant changes. Let me do the refactoring in this pr directly

Reply:
Agree.
Sorry I missed there is refine issue #2408 already, we can do the refactoring in that PR

Reply:
got it